.moai/specs/SPEC-V3R3-BRAIN-001/spec.md

```
@@ -0,0 +1,315 @@
+| 2. Diverge | Reuse `moai-foundation-thinking` Diverge-Converge — generate 5-15 angle variations | clarified_intent | divergent_concepts list (in-memory) |
+| 3. Research | Parallel WebSearch + Context7 + market scan via `moai-domain-research` skill | divergent_concepts | `research.md` |
+| 4. Converge | Reduce to 1-3 viable concepts; build Lean Canvas section | research.md + divergent_concepts | `ideation.md` (with Lean Canvas) |
+| 5. Critical Evaluation | First Principles + Critical Evaluation from `moai-foundation-thinking` | ideation.md | evaluation_report inline in `ideation.md` |
+| 6. Proposal | Final product proposal with SPEC decomposition candidate list | ideation.md + evaluation | `proposal.md` |
+| 7. Claude Design Handoff Package | Generate paste-ready prompt + curated references + acceptance criteria for claude.com Design | proposal.md + brand context (if exists) | `claude-design-handoff/{prompt.md, context.md, references.md, acceptance.md, checklist.md}` |
+
+---
+
+## 3. Output Directory Layout
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)